JSDemos/Demos/Scheduler/Editing/description.md

```diff
@@ -1 +1,13 @@
-This example demonstrates appointment editing. You can add, update and delete appointments, as&nbsp;well as&nbsp;modify them by&nbsp;dragging or&nbsp;resizing. Toggle the check boxes under the Scheduler to&nbsp;define which editing features are enabled.
\ No newline at end of file
+Scheduler allows users to add, update, resize, drag, and delete appointments. To control these operations, specify properties in the [editing](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/editing/) object. In this demo, you can use the checkboxes below the Scheduler to toggle the edit operations.
+
+These operations raise events that you can handle with the following functions:
```

Comment:
I can see only event names in the following list. I can see no functions.

Reply:
Functions that start with "on" are event handlers, e.g. onAppointmentAdding is a function, appointmentAdding is an event. [StackOverflow explanation](https://softwareengineering.stackexchange.com/questions/299101/what-does-the-prefix-on-in-a-function-e-g-onbuttonpressed-mean).